### ISSUE TITLE
Value Error 

### ISSUE THREAD

**Issue by Rohin0301** [DanaJomar/PyALE#2]
I am creating Accumulated Local Effect plots using the PyALE function. I am using a RandomForestRegression function to build the model.
 
I can create 1D ALE plots. However, I get a Value Error when I try to create a 2D ALE plot using the same model and training data.
 
Here is my code.
 
``ale(training_data, model=model1, feature=["feature1", "feature2"])``
 
I can plot a 1D ALE plot for feature1 and feature2 with the following code.
 
``ale(training_data, model=model1, feature=["feature1"], feature_type="continuous")``
 
``ale(training_data, model=model1, feature=["feature2"], feature_type="continuous")``
 
There are no missing or infinite values for any column in the data frame.
 
I am getting the following error with the 2D ALE plot command.
 
``ValueError: Input contains NaN, infinity or a value too large for dtype('float32').``
 
I am not sure why I am getting this error. I would appreciate some help on this.

I have also attached a screenshot of the stack trace of the error. 

![Stack trace](https://user-images.githubusercontent.com/50095143/117698818-bd806b80-b1e1-11eb-833f-6e15947c8d27.png)

**Comment by DanaJomar** [comment]
Hello Rohin, 
Indeed I didn't think of resetting the index of the dataframes before replacing the values of feature with the edges of the bins, which caused NAs to be generated when the index does not exist in the new data.

This issue will be fixed in an upcomming version, however in the meanwhile a simple index reset before generating the effects will do the trick.

Here is a code that reproduces the problem and the solution for it:

```python
from PyALE import ale
import pandas as pd
import matplotlib.pyplot as plt
import random
from sklearn.ensemble import RandomForestRegressor

# get the raw diamond data (from R's ggplot2)
dat_diamonds = pd.read_csv(
    "https://raw.githubusercontent.com/tidyverse/ggplot2/master/data-raw/diamonds.csv"
)
X = dat_diamonds.loc[:, ~dat_diamonds.columns.str.contains("price")].copy()
y = dat_diamonds.loc[:, "price"].copy()

features = ["carat","depth", "table", "x", "y", "z"]

# fit the model
model = RandomForestRegressor(random_state=1345)
model.fit(X[features], y)

# sample the data
random.seed(1234)
indices = random.sample(range(X.shape[0]), 10000)
sampleData = X.loc[indices, :]

# get the effects.....
# This throws the error
ale_eff = ale(X=sampleData[features], model=model, feature=["z", "table"], grid_size=100)

# This will work, just reset the index with drop=True
ale_eff = ale(X=sampleData[features].reset_index(drop=True), model=model, feature=["z", "table"], grid_size=100)

```

**Comment by DanaJomar** [comment]
Fixed in release [v1.1.2](https://github.com/DanaJomar/PyALE/releases/tag/v1.1.2)
